tools/github_reporter/package.json

```diff
@@ -14,6 +14,7 @@
     "@actions/core": "^1.10.0",
     "stack-utils": "^2.0.6"
   },
+  "type": "commonjs",
```

Comment:
CI fails without this because this folder isn’t inside `node_modules`. I already made the upstream change in https://github.com/nodejs/node/pull/49877.

Reply:
Exactly, it shouldn't fail anymore. https://github.com/GeoffreyBooth/node/commit/3f1936f698574fc9de45ec9c068b2982c317dee8 is already a part of this branch.